samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
```

Comment:
I will still call byte 0x71 to 0x1fb junk, even if it is a leftover stack.
During my checks there has not been any real data there, even if it might look like it.
The hbins and cells might have slack, but if i create a new hive, or export from an hive (cmd reg save) i expect these parts to be clean, or be an exact copy of the original which is not the case, for lack of better words I would like to call it random bytes.

To understand the internal I wrote a disector NiKiZe@1b34f83787cbe39de0f8969eab6691565a999290 running that on hive files it bacame somewhat painfully obvious. Since my goal is to create write support (or maybe a better word is build support) I want to concentrate on the used parts and not the random parts.